web/pdf_thumbnail_view.js

```diff
@@ -405,11 +423,13 @@ var PDFThumbnailView = (function PDFThumbnailViewClosure() {
     },
   };
 
+  PDFThumbnailView.cleanup = function() {
```

Comment:
You're right. Maybe I'm worried for nothing; it looks a bit strange, but it's because the one canvas is shared among `PDFThumbnailView` instances. If there would be one per instance, it would be a lot simpler, but we don't want that for memory reasons. I'm fine with this then. Sorry for the noise!